src/data/team.js

```diff
@@ -139,6 +139,17 @@ The playful, innovative work of my friend [Charlie Gerard](https://charliegerard
     gif: 'https://user-images.githubusercontent.com/3902488/153995859-eb0b6072-523e-46f5-9850-375a52876716.gif',
     alumni: false,
   },
+  {
```

Comment:
New folks get added to the top of this list. Can you move this up to the top? 🥇

Reply:
@colebemis Ah I see, added myself to the top! 🎉